src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.props

```diff
@@ -1,50 +1,18 @@
 ﻿<Project TreatAsLocalProperty="_RazorTaskFolder;_RazorTaskAssembly">
-  <!-- 
-    Properties and tasks supporting Razor MSBuild integration
-  -->
-
-  <!--
-    Default properties for common Razor SDK behavior.
-  -->
   <PropertyGroup>
     <!--
-      Set to true to automatically include Razor (.cshtml) files in @(RazorGenerate) from @(Content).
-    -->
-    <EnableDefaultRazorGenerateItems Condition="'$(EnableDefaultRazorGenerateItems)'==''">true</EnableDefaultRazorGenerateItems>
-
-    <!-- 
-      Set to true to copy RazorGenerate items (.cshtml) to the publish directory.
-
-      Typically Razor files are not needed for a published application if they participate in compilation at build-time
-      or publish-time. By default, the Razor SDK will suppress the copying of RazorGenerate items to the publish directory.
+    Used by the Web SDK to know if the Razor SDK is present. This needs to live in a nuget package (not in the SDK)
```

Comment:
> if the Razor SDK is present

This is probably my fault :sad:

Can you change this to say "if the Razor SDK can be used for compilation"

Reply:
The Razor SDK is always there. 







(watching you)